### DISCUSSION TITLE
Get timestamp value with specific timezone

### DISCUSSION THREAD

**Discussion by ShivaniSK** [finos/TimeBase-CE#287] [Q&A]
Hi @alex-karpovich ,
I am currently hitting below query to timebase

```
select from test_stream where timestamp BETWEEN '2022-08-25 00:01:00.000 America/New_York'd 
AND '2022-08-25 00:01:00.999 America/New_York'd AND SYMBOL IN ('EURUSD') 
```

Timebase has data stored in UTC currently. The above query fetches correct data based on the timezone passed. However, the timestamp returned in response is UTC epochtime only.
Is there a way to receive the response time converted from UTC to America/New_York timezone ?

Its similar to how we change the timezone in the timebase web client so that we see the response timestamp accordingly.

Regards,
Shivani

**Comment by alex-karpovich** [comment]
Greetings, @ShivaniSK 

for now we do not allow to change output timestamp field, you can have your own, like
```
select (timestamp - 4h) as localtime, field1 from test_stream where timestamp BETWEEN '2022-08-25 00:01:00.000 America/New_York'd 
AND '2022-08-25 00:01:00.999 America/New_York'd AND SYMBOL IN ('EURUSD') 
```
